ui/src/components/WelcomeCard.tsx

```diff
@@ -1,42 +1,68 @@
-import { Link } from 'react-router-dom';
-import { useLocalStorage } from 'usehooks-ts';
-
-import { createStorageKey } from '@/logic/utils';
+import { usePutEntryMutation, useSeenWelcomeCard } from '@/state/settings';
+import { useCallback, useState } from 'react';
+import { useIsMobile } from '@/logic/useMedia';
+import X16Icon from './icons/X16Icon';
 
 export default function WelcomeCard() {
-  const [isWelcomeSeen, setIsWelcomeSeen] = useLocalStorage<boolean>(
-    createStorageKey('welcome-seen'),
-    false
-  );
+  const isMobile = useIsMobile();
+  const alreadySeen = useSeenWelcomeCard();
+  const { mutate } = usePutEntryMutation({
+    bucket: 'groups',
+    key: 'seenWelcomeCard',
+  });
+  const [optimisticallyHide, setOptimisticallyHide] = useState(false);
+
+  const close = useCallback(() => {
+    setOptimisticallyHide(true);
+    mutate({ val: true });
+  }, [mutate]);
 
-  if (isWelcomeSeen) {
+  if (alreadySeen || optimisticallyHide) {
     return null;
   }
 
+  if (isMobile) {
+    return (
+      <div className="mx-6 mt-2 mb-4 space-y-4 rounded-[32px] bg-green-soft py-6 px-8">
+        <h3 className="text-lg font-medium">Welcome to Tlon</h3>
+        <p className="leading-5">
+          This is Tlon: An app for messaging friends and constructing
+          communities.
+        </p>
+        <p className="leading-5">
+          Visit the{' '}
+          <span className="font-semibold dark:font-bold">Tlon Studio</span>{' '}
```

Comment:
Might make sense to have `Tlon Studio` be a link?

Reply:
@alecananian during first time boots, this card will exist above the two pre-invited groups "Tlon Locals" and "Tlon Studio", so they'll technically be right where they need to be! That being said, I think it'd a good idea to duplicate the ability to jump to Tlon Studio in case curious people tap the bolded word.